include/deal.II/lac/block_linear_operator.h

```diff
@@ -77,7 +242,7 @@ block_operator(const std::array<std::array<LinearOperator<typename Range::BlockT
     v.reinit(m);
 
     // And reinitialize every individual block with reinit_range_vectors:
-    for (unsigned int i = 0; i < m; ++i)
+    for (size_t i = 0; i < m; ++i)
```

Comment:
The template parameter ```m``` is a ```size_t``` - which is necessary because ```std::array``` has a ```size_t``` as argument. Otherwise, some template deduction fails.